Scripts/BuildGVFSForWindows.bat

```diff
@@ -15,23 +15,32 @@ IF NOT EXIST %nuget% (
 )
 
 :: Acquire vswhere to find dev15 installations reliably.
-SET vswherever=2.5.2
+SET vswherever=2.6.7
 %nuget% install vswhere -Version %vswherever% || exit /b 1
 SET vswhere=%VFS_PACKAGESDIR%\vswhere.%vswherever%\tools\vswhere.exe
 
-:: Use vswhere to find the latest VS installation (including prerelease installations) with the msbuild component.
+:: Use vswhere to find the latest VS installation with the msbuild component.
 :: See https://github.com/Microsoft/vswhere/wiki/Find-MSBuild
-for /f "usebackq tokens=*" %%i in (`%vswhere% -all -prerelease -latest -version "[15.0,16.0)" -products * -requires Microsoft.Component.MSBuild Microsoft.VisualStudio.Workload.ManagedDesktop Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Workload.NetCoreTools Microsoft.Component.NetFX.Core.Runtime Microsoft.VisualStudio.Component.Windows10SDK.10240 -property installationPath`) do (
+for /f "usebackq tokens=*" %%i in (`%vswhere% -all -prerelease -latest -products * -requires Microsoft.Component.MSBuild Microsoft.VisualStudio.Workload.ManagedDesktop Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Workload.NetCoreTools Microsoft.Net.Core.Component.SDK.2.1 -property installationPath`) do (
   set VsInstallDir=%%i
 )
 
+:: Assumes default installation location for Windows 10 SDKs
+IF NOT EXIST "c:\Program Files (x86)\Windows Kits\10\Include\10.0.10240.0" (
+  echo ERROR: Could not find Windows 10 SDK Version 10240

```

Comment:
Can we point the user to the readme like we do in the "can't find VS" error message?